File: Python/temp.py
import math
import sys
from typing import Counter, List, Optional


class NumArray:

    num = [0]*200000

    def __init__(self, nums: List[int]):
        for i in range(len(nums)):
            self.num[i+1] = self.num[i]+nums[i]

    def sumRange(self, left: int, right: int) -> int:
        return self.num[right+1]-self.num[left]

File: Java/Solution/src/temp.java
public class temp {
    public static void main(String[] args) {
        int num[][] = new int[3][3];
        for (int i = 0; i < 3; i++) {
            for (int j = 0; j < 3; j++) {
                num[i][j] = i * j;
            }
        }
        int temp[][] = num;
        temp[0][0] = 1;
        System.out.println(num[0][0]);
    }
}
